internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -393,7 +393,7 @@ func ParseSession(s Session) (sdktypes.Session, error) {
 
 type Deployment struct {
 	DeploymentID string `gorm:"primaryKey"`
-	EnvID        string `gorm:"foreignKey"`
+	EnvID        string `gorm:"index;foreignKey"`
 	BuildID      string `gorm:"foreignKey"`
```

Comment:
if you are going to query deployments by build id, you will need an index as well

Reply:
no, we are not querying deployments by buildID. The opposite - gathering buildIDs by deployments